server/planning/content_api/events/__init__.py

```diff
@@ -8,7 +8,47 @@
 # AUTHORS and LICENSE files distributed with this source code, or
 # at https://www.sourcefabric.org/superdesk/license
 
+from superdesk.flask import g
+from eve.utils import ParsedRequest
 from superdesk.core.module import Module
-from .event import content_api_event_resource_config
+from superdesk.core.web import EndpointGroup
+from werkzeug.datastructures import MultiDict
+from superdesk.core.types import Request, Response
+from .event import content_api_event_resource_config, ContentAPIEventService
 
-module = Module("planning.content_api.events", resources=[content_api_event_resource_config])
+event_endpoints = EndpointGroup("events_capi", __name__)
+
+
+@event_endpoints.endpoint("event", methods=["GET"])
+async def get_event_list(args, params, request: Request) -> Response:
+    service = ContentAPIEventService()
+    req = ParsedRequest()
+    req.args = MultiDict()
```

Comment:
It would be good if we could add support here for the allowed params, at the moment none are passed through from this endpoint to the underlying service